backend/dal/Migrations/20201021213937_v01.01.00.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010100 : Migration
```

Comment:
You will need to inherit from `SeedMigration` so that you can update the db with the new role and claims.